pipelines/cdp/scripts/02_run_dataflow_job.sh

```diff
@@ -0,0 +1,18 @@
+python3 -m main \
+  --runner=DataflowRunner \
+  --project=$PROJECT \
+  --temp_location=gs://$PROJECT/tmp \
+  --region=$REGION \
+  --save_main_session \
+  --service_account_email=my-dataflow-sa@$PROJECT.iam.gserviceaccount.com \
```

Comment:
The name of the service account should be populated by Terraform in an environment variable. This is a non-blocking comment, you can submit that in a new pull request after this one is merged.